utils/jest/preprocessor.js

```diff
@@ -0,0 +1,22 @@
+const mergeAll = require('ramda').mergeAll;
+const babel = require('../../config/babel')();
+const babelJest = require('babel-jest');
+const buildConfigs = require('../buildConfigs');
+const config = require('../kytConfig')();
+
+const { clientConfig } = buildConfigs(config);
+
+// Merge userland babel config with our babel config
+// TODO: This logic should probably move to config/babel.js,
+// but that would be a breaking change.
+const clientBabelConfig = clientConfig.module.loaders
+  .find(loader => loader.loader === 'babel-loader')
+  .query;
+
+// TODO: What's sort of confusing here is that userland babel config
+// (at least for Project-Vi) does not put things in babel.env's.
```

Comment:
I know this is kind of a small thing, but can we leave references to internal NYT tools outside of kyt now that it's open? Outside users won't understand what we're comparing against. Possibly even make this an issue instead of a comment/aside?